google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -246,6 +246,16 @@ protected void resetEditorFrom(@NotNull AppEngineDeploymentConfiguration configu
     }
     archiveSelector.setText(configuration.getUserSpecifiedArtifactPath());
 
+    if (configuration.isStagedArtifactNameLegacy()) {
```

Comment:
if it is "legacy" don't we also want to populate the UI field with placeholder text showing the name we are going to use (e.g. _.jar/war)?

Reply:
By setting the configuration's staged artifact name appropriately, the field's text shows this legacy value `target.jar/war` (see line 260 below):

```
stagedArtifactNameTextField.setText(configuration.getStagedArtifactName());
```